app.rb

```diff
@@ -1,43 +1,118 @@
 require './book'
-require './teacher'
 require './student'
+require './teacher'
 require './rental'
 require './person'
+require './checks'
+require_relative 'create_book'
+require_relative 'create_person'
+require './create_rental'
+require 'json'
 
 class App
-  attr_reader :books, :people, :rentals
+  include Checks
+  attr_accessor :books, :people
 
   def initialize
-    @books = []
     @people = []
-    @rentals = []
+    @books = []
+    @rents = []
+    @create_book = CreateBook.new(@books)
+    @create_person = CreatePerson.new(@people)
+    @create_rental = CreateRental.new(@rents, @books, @people)
   end
 
-  def list_of_books
-    @books
+  def run(choice)
+    case choice
+    when 1
+      list_books
+    when 2
+      list_people
```

Comment:
- [ ] I'm not able to see the list of books and people, When I select the first or the second option, the app shows the following error:
<img width="548" alt="image" src="https://user-images.githubusercontent.com/77190203/168926738-d2b8dc65-31f0-4b16-bae3-7cd84688920d.png">

Following the project requirements, Please make sure:
  - After closing the app, the data should be saved (all the data: books, people, and rentals) into JSON files. One per type of data should be stored and named accordingly (e.g. rentals.json)
This is an example of people.json:
```json
[{"json_class":"Student","name":"Sam","age":"14","parent_permission":true},{"json_class":"Student","name":"Pom","age":"16","parent_permission":false},{"json_class":"Teacher","name":"Fox","age":"56","parent_permission":true,"specialization":"Starships"},{"json_class":"Student","name":"Poppy","age":"20","parent_permission":false}]
```
  - On the startup of your app the data should be loaded from the JSON files. The app should work and show no errors

Reply:
Hi, @medaminedev66  can you check again, please? The exit option in my menu starts with uppercase: `7.Exit` and on your screen, it appears in lowercase `7.exit`, maybe this mistake is from another repo.
![Screenshot_1](https://user-images.githubusercontent.com/92594336/168937191-0a22e75c-c4b4-4a82-b3bb-465838160ed2.png)
 
I ran again my project and all is working fine.

Reply:
Hi @yersel500, I will check out your repo again.

Reply:
Hi @yersel500, when I cloned your repository, I checked out the  `preserve-data` branch instead of the `saving-files` branch. Sorry for this mistake. I tested your app on the branch `saving-files` and it works well and has no errors. You can request another review and your pull request will be approved immediately.